api/server/server.go

```diff
@@ -1115,6 +1124,11 @@ func (s *Server) bindHandlers(ctx context.Context) {
 			v2.DELETE("/triggers/:triggerID", s.handleTriggerDelete)
 		}
 
+		if !s.noCallEndpoints {
+			v2.GET("/fns/:fnID/calls", s.handleCallList2)
```

Comment:
Should the server provide a helpful "these are disabled" message if we have an API server which is not enabling these endpoints.